pkgs/applications/networking/cluster/taktuk/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, perl , openssh}:
+
+stdenv.mkDerivation rec {
+  version = "3.7.5";
+  name = "taktuk-${version}";
+
+  buildInputs = [ perl ];
+
+  src = fetchurl {
+    url = "https://gforge.inria.fr/frs/download.php/33412/taktuk-3.7.5.tar.gz";
```

Comment:
OK, for ${name}.
But for the build, it's weird:
- perl is already in the buildInputs
- It builds ok for me, and the resulting perl script uses the good perl binary:
```
    bzizou@killeen:~/nixpkgs$ head -1 $(which taktuk)                      
    #!/nix/store/s5hmy8kv1w151a1z4wy57csla3c11jr9-perl-5.20.3/bin/perl -w 
```
And during the build, I can see:
`checking for perl... /nix/store/s5hmy8kv1w151a1z4wy57csla3c11jr9-perl-5.20.3/bin/perl`

I can patch, but why in your case does the build uses /usr/bin/perl?